Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -1026,7 +1026,10 @@ - (void)handleElementActionWithType:(_WKElementActionType)type element:(_WKActiv
         [delegate actionSheetAssistant:self performAction:WebKit::SheetAction::Copy];
         break;
     case _WKElementActionTypeOpen:
-        [delegate actionSheetAssistant:self openElementAtLocation:element._interactionLocation];
+        if (element.URL)
+            [[UIApplication sharedApplication] openURL: element.URL withCompletionHandler:nil];
```

Comment:
Does this open web links in a new tab?

Reply:
Also, if we're using `openURL:`, we should probably use the API variant `openURL:options:completionHandler:`.

This SPI usage is causing a build failure, and I don't think we need to use the SPI here.

Reply:
> Does this open web links in a new tab?

No

> Also, if we're using openURL:, we should probably use the API variant openURL:options:completionHandler:.

Ah ok, I was just following the advice that @whsieh suggested

Reply:
(On a related note, it might also be worth cleaning up the existing call site in `-contextMenuInteraction:willPerformPreviewActionForMenuWithConfiguration:animator:` at some point)